bindings/psat.h

```diff
@@ -56,31 +57,34 @@ NAN_METHOD(headToolSuctionTank) {
     const double dischargeGaugeElevation = Get("dischargeGaugeElevation");
     const double dischargeLineLossCoefficients = Get("dischargeLineLossCoefficients");
 
-    HeadToolSuctionTank htst(specificGravity, flowRate, suctionPipeDiameter, suctionTankGasOverPressure,
-                             suctionTankFluidSurfaceElevation, suctionLineLossCoefficients, dischargePipeDiameter,
-                             dischargeGaugePressure, dischargeGaugeElevation, dischargeLineLossCoefficients);
+    ;
+
+    auto rv = HeadToolSuctionTank(specificGravity, flowRate, suctionPipeDiameter, suctionTankGasOverPressure,
+        suctionTankFluidSurfaceElevation, suctionLineLossCoefficients, dischargePipeDiameter,
+        dischargeGaugePressure, dischargeGaugeElevation, dischargeLineLossCoefficients).calculate();
+
 
-    auto rv = htst.calculate();
     Local<String> differentialElevationHead = Nan::New<String>("differentialElevationHead").ToLocalChecked();
```

Comment:
Need to remove all instances of Local<String> [name]
These initialize the strings used to establish keys for the key-value pairing, but now we are entering the string directly into the SetR function. These declarations are no longer necessary.